cloudferry/model/__init__.py

```diff
@@ -15,26 +15,9 @@
 Model module contain tools to define cloud object schema and a way to store
```

Comment:
i think using something like `base.py` module instead of putting the code into `__init__.py` is better.